web/src/components/search/SpeciesSearch.js

```diff
@@ -21,20 +21,20 @@ const useStyles = makeStyles(({ palette, typography }) => ({
       '& .MuiTableHead-root': {
         '& .MuiTableRow-head': {
           '& .MuiTableCell-head': {
-            fontSize: typography.table.fontSize,
```

Comment:
needed for tests since we aren't applying a theme and don't care about styling for the tests.